minecraft_servers/hardwarecraft/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "server_name": "hardwarecraft",
+  "nice_name": "Hardware Craft",
+  "direct_ip": "hardware-craft.xyz",
+  "server_wildcards": [
+    "%.hardware-craft.xyz",
+    "%.hardware-network.de"
+  ],
+  "social": {
+    "web": "https://hardware-network.de",
+    "web_support": "https://events.hardware-network.de",
```

Comment:
Website is in maintenance - Please remove or update. Also it looks like a event announcement page, not like a support page.

Reply:
I removed the support website, is it all perfect now?